### ISSUE TITLE
Generate posts not working

### ISSUE THREAD

**Issue by datlechin** [migratetoflarum/fake-data#18]
```
POST http://flarum.test/api/fake-data

PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '6-1' for key 'posts.posts_discussion_id_number_unique' in
```

**Comment by clarkwinkelmann** [comment]
This might related to the Flarum 1.3 changes to post numbering. I'm not actually sure I have tested the extension on Flarum 1.3 yet. Will try to fix this in the next few days.

**Comment by clarkwinkelmann** [comment]
Fixed in version 1.3.0 of the extension!
